src/patternfly/components/LabelGroup/examples/LabelGroup.md

```diff
@@ -0,0 +1,402 @@
+---
+id: 'Label group'
+beta: true
+section: components
+cssPrefix: pf-c-label-group
+---
+
+## Examples
+### Basic
+```hbs
+{{#> label-group label-group--id="label-group-basic"}}
+  {{#> label-group-list label-group-list--attribute='aria-label="Label group list"'}}
+    {{#> label-group-list-item}}
+      {{#> label}}
+        {{#> label-icon}}
+          <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+        {{/label-icon}}
+        Label
+      {{/label}}
+    {{/label-group-list-item}}
+    {{#> label-group-list-item}}
+      {{#> label label--modifier="pf-m-blue"}}
+        {{#> label-icon}}
+          <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+        {{/label-icon}}
+        Label 2
+      {{/label}}
+    {{/label-group-list-item}}
+    {{#> label-group-list-item}}
+      {{#> label label--modifier="pf-m-green"}}
```

Comment:
should the last label group item have no margin-right
<img width="330" alt="Screen Shot 2020-08-18 at 1 18 25 PM" src="https://user-images.githubusercontent.com/20118816/90544660-7505b700-e155-11ea-8a64-9705f34dbcab.png">

Reply:
Nope, there is a negative right margin on the container. That allows labels in the middle of the list to touch the right edge of the container when the labels wrap.